payment/order.go

```diff
@@ -140,3 +143,61 @@ func CreateOrderItemFromMacaroon(sku string, quantity int) (*OrderItem, error) {
 func (order Order) IsPaid() bool {
 	return order.Status == "paid"
 }
+
+// IsStripePayable returns true if every sku is payable by Stripe
+func (order Order) IsStripePayable() bool {
+	return true
```

Comment:
This function will check for the new Brave Together paid SKU

Reply:
Are we going to have a list of SKUs that are stripe payable?  Should there be a flag in the macaroon stating valid payment options?